### ISSUE TITLE
[PSX][Octoshock+Nymashock][Gex - Enter the Gecko] BizHawk 2.9.1 locks up when attempting to load Gex - Enter the Gecko for PSX for both included cores

### ISSUE THREAD

**Issue by boktai1000** [TASEmulators/BizHawk#3863]
[//]: # "This description supports Markdown syntax. There's a cheatsheet here: https://guides.github.com/features/mastering-markdown/"
[//]: # "These lines are comments, for letting you know what you should be writing. You can delete them or leave them in."
[//]: # "Also, please don't waste your time writing until you've checked for similar Issues. Remember to check closed Issues too!"
[//]: # "One more thing: if you're on Linux, please comment on #1430 instead of opening an issue so we don't annoy the other devs."

### Summary
[//]: # "Briefly describe what's broken. Include relevant details: loaded core, loaded rom's hash, open tools, running scripts... You can embed a screenshot if it's easier to show the bug, but if you need more than one please put them at the end."

When attempting to open / run Gex - Enter the Gecko for PS1, BizHawk will completely lock up.

![image](https://github.com/TASEmulators/BizHawk/assets/9054563/0601cb66-8c17-47ad-980f-fbc56771412b)

```
Algorithm : MD5
Hash      : 68F3483C474B8582D985FD6283FEC66B
File      : Gex - Enter the Gecko (USA).cue

Algorithm : MD5
Hash      : 078092D2434011A12D5C56DA4DE0EEB4
File      : Gex - Enter the Gecko (USA).bin

Algorithm : MD5
Hash      : 490F666E1AFB15B7362B406ED1CEA246
Path      : PSX_3.0(A).bin
```

Additionally afterwards I have installed the Libretro PSX core following steps here that I used to try some of my CHD formatted games, and can confirm that this core/configuration did work and load the game properly for what it's worth.
https://www.reddit.com/r/EmulationOnPC/comments/11pj0u3/comment/jc1fjfl/

I have also tested these bin/cue files with DuckStation and RetroArch and can confirm they loaded without issue there.

### Repro
[//]: # "If you can't figure out the list of steps, delete this section and put 'heisenbug' in the summary somewhere. If a Lua script can cause the bug, you can embed that instead (as simple as possible please)."
1. Setup BizHawk with firmware required for PS1.
2. Attempt to load a Redump verified copy of Gex - Enter the Gecko for PS1 using defaults / default core, or swapping to other PS1 core (both Octoshock and Nymashock exhibit lockup behavior)
### Output
[//]: # "Paste the contents of the error dialog if there is one (try Ctrl+C, it usually works), or paste the output from the Lua Console, or delete this section."

Unfortunately there is no Output even from having the Log opened beforehand.

![image](https://github.com/TASEmulators/BizHawk/assets/9054563/0c8a0792-62a5-466a-b5bc-7901e142acc5)

### Host env.
[//]: # "List the computers you've found the bug with. If there's a version that doesn't have the bug, please put that in too. Here are some examples:"
- BizHawk 2.91 - Windows 10 x64 (22H2)

[//]: # "(screenshots, if applicable)"

[//]: # "That's it! If you'd like to help more, you could try a dev build (see Testing in the readme) or an older release. Click submit now and you can edit it later."

**Comment by YoshiRulz** [comment]
CNR with 2.9.1 or a3f801548. My `.bin` and firmware are the same as yours, and I could match your `.cue` by renaming the file.

**Comment by boktai1000** [comment]
Huh, you're right. I downloaded a fresh copy of BizHawk and made a test folder and just put the Firmware there and was able to open it successfully. My apologies there - do you have any suggestions on where I should take troubleshooting or next steps? I understand if this should be closed, but I'd be curious to get to the bottom of this, as other PS1 games work with my configuration, so something else must be off.

Thanks for the verification!

**Comment by YoshiRulz** [comment]
Copy `config.ini` into the fresh copy and I expect it will stop working there too.

**Comment by boktai1000** [comment]
> Copy `config.ini` into the fresh copy and I expect it will stop working there too.

Confirmed crashing with my setups `config.ini`

I was about to upload my configure but before doing so, I logged out of RetroAchievements to make sure that information didn't get passed through. It seems it's something related with RA, because after signing out - the game loaded and worked as expected.

![image](https://github.com/TASEmulators/BizHawk/assets/9054563/a67e3858-de69-4251-b041-f99be813c1cb)

Note: I redacted my username from both files, and then also my RetroAchievements username & token, but the rest is unmodified.

[WORKING_config.zip](https://github.com/TASEmulators/BizHawk/files/14466810/WORKING_config.zip)

[NOTWORKING_config.zip](https://github.com/TASEmulators/BizHawk/files/14466816/NOTWORKING_config.zip)

**Comment by CasualPokePlayer** [comment]
Only meaningful difference between the two configs is one has RetroAchievements active and the other does not, quite strange. I could guess maybe it's something with the RA specific hashing locking up?

**Comment by CasualPokePlayer** [comment]
I can't reproduce any issues with RetroAchievements on the latest dev build, although I can on 2.9.1. I'll assume this got fixed at some point after 2.9.1.
